### PR TITLE
Ivan

### PR DESCRIPTION
**Pull request by kinca3003**
Test example

### CODE REVIEW

**Comment by Suna898789** [comment]
Hi Ivan! Please would you rename the package week2  you did write it week_2 and that will cause problem in the structure of the project  when merging. 
Please just write "week2" without underscore "week_2"

**Comment by kinca3003** [comment]
Hi, ok!


вт, 27 февр. 2024 г. в 10:33, Suna898789 ***@***.***>:

File: src/Ivan/week1/Divide.java
Comment (Suna898789) [inline src/Ivan/week1/Divide.java:1]: good

File: src/Ivan/week1/Finra.java
Comment (Suna898789) [inline src/Ivan/week1/Finra.java:1]: good

File: src/Ivan/week1/Numbers.java
Comment (Suna898789) [inline src/Ivan/week1/Numbers.java:1]: good

File: src/Ivan/week1/Tasks.txt
Comment (Suna898789) [inline src/Ivan/week1/Tasks.txt:1]: good

File: src/Ivan/week3/ReverseNegativeNumber.java
Comment (Suna898789) [inline src/Ivan/week3/ReverseNegativeNumber.java:1]: good

File: src/Ivan/week_2/Swap.java
Comment (Suna898789) [inline src/Ivan/week_2/Swap.java:1]: good

**Review by Suna898789 — APPROVED** [review]

**Pull request merged**